pkgs/by-name/ta/tabby-terminal/package.nix

```
@@ -0,0 +1,242 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchurl,
+  fetchYarnDeps,
+  nodejs,
+  yarn,
+  git,
+  patch-package,
+  electron_29,
+  prefetch-yarn-deps,
+  fixup-yarn-lock,
+  python3,
+  makeWrapper,
+  http-server,
+  fontconfig,
+  pkg-config,
+  libsecret,
+  makeDesktopItem,
+  copyDesktopItems,
+}:
+
+let
+  tabby = rec {
+    pname = "tabby-terminal";
+    version = "1.0.215";
+
+    src = fetchFromGitHub {
+      owner = "Eugeny";
+      repo = "tabby";
+      tag = "v${version}";
+      hash = "sha256-l3Hyt3e1edJjb7vL66JDllkpHwTU5efsNbgJ5Pcvn60=";
+      leaveDotGit = true;
+      deepClone = true;
+    };
+
+    meta = {
+      description = "A terminal for a more modern age";
+      homepage = "https://tabby.sh";
+      license = lib.licenses.mit;
+      mainProgram = "tabby";
+      maintainers = with lib.maintainers; [ geodic ];
+      platforms = [
+        "x86_64-linux"
+        "aarch64-linux"
+      ];
```

Leave it general unless there's specific reasons against it
```suggestion
      platforms = lib.platforms.linux;
```